build_matrix.yml

```diff
@@ -205,12 +207,7 @@ matrix:
     - ruby: "2.7.7"
     - ruby: "3.0.4"
     - ruby: "3.1.2"
-    - ruby: "3.2.0-rc1"
-      env_vars:
-        - name: "_LIBYAML"
-          value: "1"
```

Comment:
Removing this means we can also remove these lines: https://github.com/appsignal/appsignal-ruby/blob/7f62ada8deb67a2b7d355ec0c1bc2ad1d1e2d8d1/build_matrix.yml#L28-L33